cfgs/config/advanced.cfg

```diff
@@ -1266,12 +1266,11 @@ INCLUDE gedcolor.cfg
 // Spelling checker default language.
 // The numberic code is specifying for Microsoft Spellchecker
 // (US english - 1033, Russian - 1049, another see into Microsoft's documentation).
-// The case-sencitive dictionnary name is specifying for MySpell (file name
-// without suffixes ".dic" or ".aff").
+// The case-sencitive dictionnary space separated names are specifying for MySpell
```

Comment:
typo: sensitive

Reply:
This is still bad English. I don't understand what both you and the previous author wanted to say. Maybe it would be better if you write all these sentences in Russian and then translate them in English using translate.yandex.ru.

Reply:
Yep, not the best English ever :)
I can try to rewrite it when I have some time. I added multiple dictionaries support long time ago, so it looks like it's my duty to update sample config description accordingly...

Reply:
I think it worth moving English language fixes into a separate PR, don't block this one with comments spelling issues.

Reply:
@vitaliy-aksyonov:  I can quite understand what you have written. With some small corrections made by LanguageTool, it looks like that.

The meaning of this keyword is the list of dictionaries to use by default. Space separated. Each dictionary is a case-sensitive dictionary filename without extension. For example, ru_RU for ru_RU.{dic,aff}.

@spanevin: Besides translate.yandex.ru, you may use the LanguageTool (https://languagetool.org/). It is free of charge and they have a browser extension.

Reply:
Thanks. I'll make those changes.

Reply:
@dukelsky thanks for the recommendations.

Reply:
Updated comment. @dukelsky, please review.